docs/src/token.md

```diff
@@ -2,9 +2,76 @@
 title: Token Program
 ---
 
-An ERC20-like Token program on the Solana blockchain.
+A Fungible Token program on the Solana blockchain.
 
-The project comprises:
+This program provides an interface and implementation that third parties can use to create and trade their tokens.
 
-* The Rust on-chain program
-* A JavaScript library to interact with the on-chain program
+## Background
+
+Solana's programming model and the definitions of the Solana terms used in this document are available at:
+- https://docs.solana.com/apps
+- https://docs.solana.com/terminology
+
+## Source
+
+The Token Program's source is available on [github](https://github.com/solana-labs/solana-program-library)
+
+## Interface
+
+The on-chain Token Program is written in Rust and available on crates.io as [spl-token](https://docs.rs/spl-token/1.0.2/spl_token/) documents the program's [instruction interface documentation](https://docs.rs/spl-token/1.0.2/spl_token/instruction/enum.TokenInstruction.html)
```

Comment:
`..as [spl-token](https://docs.rs/spl-token/1.0.2/spl_token/) documents..` -- something's wrong with the sentence here